question16/main.js

```diff
@@ -41,35 +41,34 @@ async function init() {
 }
 init();
 
-function renderNewsUiElement(data) {
-  tabsGroup.appendChild(createTabs(data));
-  newsWrapper.appendChild(createContents(data));
+function renderNewsUiElement(newsData) {
+  tabsGroup.appendChild(getTabListsFragment(newsData));
+  newsWrapper.appendChild(renderContents(newsData));
 }
 
-//Render Tab Element
-const createTabs = (data) => {
+//get Tab Elements 
+const getTabListsFragment = (newsData) => {
   const fragmentTablists = document.createDocumentFragment();
-  for (let i = 0; i < data.length; i++) {
+  for (let i = 0; i < newsData.length; i++) {
     const tabList = createElementWithClassName("li", "tabList");
-    tabList.textContent = data[i].category;
-    i === 0 && tabList.classList.add("is-active-tab");
+    tabList.textContent = newsData[i].category;
+    newsData[i].initialDisplay === true && tabList.classList.add("is-active-tab");
```

Comment:
IMO
It can be omitted.
```suggestion
    newsData[i].initialDisplay && tabList.classList.add("is-active-tab");
```

Reply:
Thank you for your review! I learned a lot! I've corrected it as you suggested!